src/main/scala/net/snowflake/spark/snowflake/SnowflakeTelemetry.scala

```diff
@@ -59,49 +59,40 @@ object SnowflakeTelemetry {
     }
   }
 
-  private var cachedSparkLibraries: Seq[String] = Seq.empty
```

Comment:
change 2:
Remove the spark libraries caching. I introduced the caching for better performance. But it may cause the actual libraries not to be collected.